bazel/init/stage_1.bzl

```diff
@@ -276,3 +276,16 @@ def stage_1():
         ],
         patch_args = ["-E", "-p1"],
     )
+
+    maybe(
+        name = "dropbear",
+        repo_rule = http_archive,
+        build_file = "//bazel/dependencies:dropbear.BUILD.bazel",
+        sha256 = "e02c5c36eb53bfcd3f417c6e40703a50ec790a1a772269ea156a2ccef14998d2",
+        urls = ["https://github.com/mkj/dropbear/archive/refs/tags/DROPBEAR_2022.83.tar.gz"],
+        strip_prefix = "dropbear-DROPBEAR_2022.83",
+        patches = [
+            "//bazel/dependencies/dropbear:0000-allow-blank-password.patch",
```

Comment:
Could we set the user password in the test VM to blank instead?  This would let us run unpatched dropbear (and simplify future updates)

Reply:
We're using 9p in many cases, we don't have a dedicated image for the test. (which has a lot of benefits).
Also, the option being here does not mean we're using it. Eg, in cases we have a full custom image -> yes, let's setup authentication properly in the image itself.